hw/ip/otp_ctrl/data/otp_ctrl.hjson

```diff
@@ -291,6 +291,18 @@
       default: "4",
       local: "true"
     },
+    { name: "CreatorSwCfgRetRamResetMaskOffset",
```

Comment:
do we think this is a creator otp configuration..? in a way it almost feels owner to me.

Reply:
I thought so, we also briefly talked about this in the mask rom meeting and we thought this should be under creator's control.

Reply:
sounds good.